source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -16,7 +16,7 @@
 
 open Core
 open Data_structures
-module Json = Yojson.Safe
+module JsonAst = JsonParsing.JsonAst.Json
```

Comment:
I would prefer if `JsonAst = JsonParsing.JsonAst`.
If someone doesn't see that line, they would wonder what `JsonAst.t` is since it's in fact `JsonAst.Json.t`.
There are also a few places where we use `JsonParsing.JsonAst.Location.t`, it would be shorter with `JsonAst.Location.t`.
This can be fixed in a different PR.